apps/condo/domains/resident/schema/SuggestServiceProviderService.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Generated by `createservice organization.SuggestServiceProviderByTinService --type queries`
+ */
+
+const { pick, uniqBy } = require('lodash')
+
+const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { MeterResourceOwner } = require('@condo/domains/meter/utils/serverSchema')
```

Comment:
Please avoid using server utils, when possible. They're doing full gql request emulation, but skipping access.

So your request becomes very heavy by all relations from domain/X/gql.js

Reply:
Use `find` / `getByCondition` and so on